test/functional/ios/webview/basics-specs.js

```diff
@@ -38,26 +38,18 @@ describe('webview - basics', function () {
       });
     }).nodeify(done);
   });
-  it('setting context to \'WEBVIEW_1\' should work', function (done) {
-    driver
-      .contexts().should.eventually.have.length.above(0)
-      .context("WEBVIEW_1")
-      .sleep(500)
-      .get(env.GUINEA_TEST_END_POINT)
-      .sleep(500)
-      .title()
-      .should.eventually.equal("I am a page title")
-      .nodeify(done);
-  });
-  it('setting context to \'WEBVIEW_1\' should work without first getting contexts', function (done) {
```

Comment:
By getting rid of this test are we admitting that the only correct pattern for changing contexts is to get the contexts first?  For tests on a simulator it works to hard code the context, so this _could_ just be sequestered from real device tests, and we would know that we could, should we want to, set a context without first getting the list.